### ISSUE TITLE
':' is missing in the code which is why the aria-label displaying trans('shop::app.customers.account.addresses.create.country') at create address page on inspect getting issue in testing

### ISSUE THREAD

**Issue by kartikeywebkul9260** [bagisto/bagisto#10414]
**Describe the bug**
 - Branch [master]
at /packages/Webkul/Shop/src/Resources/views/customers/account/addresses/create.blade.php
                        <x-shop::form.control-group.control
                            type="select"
                            name="country"
                            rules="{{ core()->isCountryRequired() ? 'required' : '' }}"
                            v-model="country"
                            aria-label="trans('shop::app.customers.account.addresses.create.country')"
                            :label="trans('shop::app.customers.account.addresses.create.country')"
                        >
     ':' is missing in the code which is why the aria-label displaying trans('shop::app.customers.account.addresses.create.country') at create address page on inspect getting issue in testing

**Screenshots**
![Screenshot from 2024-12-19 14-17-46](https://github.com/user-attachments/assets/9cfcf31e-0a69-41e0-aa48-9a4b76a7a694)
![Screenshot from 2024-12-19 13-56-45](https://github.com/user-attachments/assets/8734f54a-a6f5-449f-a5ea-dd148d3c3e39)
